client/public/static/js/account/update_recipe.js

```diff
@@ -0,0 +1,153 @@
+$(document).ready(function () {
+  $("#updateRecipeForm").on("submit", async function (event) {
+    event.preventDefault();
+    const username = utils.getUserNameFromCookie();
+    const userId = await utils.getUserIdFromUsername(username);
+    let formRecipeName = document.getElementById('recipeName');
+    
+    let urlAction = "";
+    let request = {};
+    let successMessage = "";
+    let errorMessage = "";
+
+    urlAction = DELETE_ACTION;
+    request = {
+      recipeName: formRecipeName.value
+    }
+    successMessage = SUCCESSFULLY_DELETED_RECIPE;
+    errorMessage = UNABLE_TO_DELETE_RECIPE_ERROR;
+    
+    let url = `${USER_FAVORITES_RECIPES_CRUD_URL}/${userId}/recipe/delete_recipe`;
+    console.log(`Sending [${urlAction}] request to: ${url}`)
+
+    fetch(url, {
```

Comment:
Are we handling delete from the update form and the recipe details view? I see the url is the delete endpoint

Reply:
Yea so what I ended up doing is due to not being able to create a new recipe with the same name as a currently existing recipe, when the user clicks the update recipe button the original recipe is actually deleted and a "new" recipe is added in its place.

Reply:
So what is the flow for the user if they go to update a recipe?
Recipe details -> update button -> update recipe form?

If we select the update and delete the recipe before submitting the form what would happen if the user just hits the back button. Is the recipe technically deleted? 

Prob want to have two diff endpoints for create/update imo. What do you think?

Reply:
My account -> my recipes -> recipe (click update recipe) -> brings them to a page that looks just like create recipe with all content brought over - besides images currently -> update recipe deletes old and saves new.

Recipe is not deleted until either delete button is clicked on actual recipe or update is clicked on update recipe page.

Yea, that is fine. I figured it was same functionality and would just be more overhead, but I understand the reasoning for it.

Reply:
Ahh ok well then that's not too bad! Prob better to still keep the endpoints separated better practice instead of deleting and then updated. I could see in future querying based on the object id so we could have multiple recipes from different users created.